lib/pbench/test/functional/server/conftest.py

```diff
@@ -17,7 +17,7 @@ def pbench():
     assert (
         host
     ), "Pbench Server functional tests require that PBENCH_SERVER be set to the hostname of a server"
-    pbench = Pbench(host)
-    assert pbench, f"Unable to connect to Pbench Server {host}"
-    pbench.connect()
-    return pbench
+    pbench_client = PbenchServerClient(host)
+    assert pbench_client, f"Unable to connect to Pbench Server {host}"
+    pbench_client.connect({"accept": "application/json"})
+    return pbench_client
```

Comment:
I left the `assert` as a black-box check. However failure in the constructor ought to be an exception, which would fail the test, even if the constructor *could* fail, and since it's not connecting the message isn't particularly relevant. 😦

Reply:
Just so.